static/js/src/certified-search-results.js

```diff
@@ -141,8 +142,19 @@ function toggleShowAllLinks() {
   });
 }
 
+function hideDrawerPageReload() {
+  const applyFilters = document.querySelectorAll(".js-apply-filters");
+  applyFilters.forEach((button) => {
```

Comment:
Are we using this argument anywhere? If not maybe we should remove it. If we had linting, this probably would be highlighted

Reply:
Good spot - I thought I was going to add an event listener but then realised I just want it to hide on page reload so didn't need it. removed.